csrankings-k.csv

```diff
@@ -530,7 +530,7 @@ Kento Aida,National Institute of Informatics,https://www.nii.ac.jp/en/faculty/ar
 Keqin Li 0001,Hunan University,http://csee.hnu.edu.cn/people/likeqin,x0YtT7QAAAAJ
 Keren Censor,Technion,http://ckeren.net.technion.ac.il,HdOuwlQAAAAJ
 Keren Censor-Hillel,Technion,http://ckeren.net.technion.ac.il,HdOuwlQAAAAJ
-Keren Zhou 0001,George Mason University,https://cs.gmu.edu/directory/detail/175,5qCK-R0AAAAJ
+Keren Zhou 0001,George Mason University,https://www.jokeren.tech,5qCK-R0AAAAJ

```

Comment:
Please confirm that this faculty member has a 75%+ appointment at GMU.

Reply:
@emeryberger yes, Keren is full time tenure track assistant prof. at GMU.  He was at OpenAI before joining GMU in Spring'24.